creating-joy/src/main/java/com/technocrats/creatingjoy/controller/AuthController.java

```diff
@@ -1,4 +1,83 @@
 package com.technocrats.creatingjoy.controller;
 
+import com.technocrats.creatingjoy.dto.UserDTO;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
 public class AuthController {
-}
+
+    List<UserDTO> users = new ArrayList<UserDTO>();
+
+    @GetMapping("/")
+    public String login() {
+        return "login_signup";
+    }
+
+
+    @GetMapping("/showLogin")
+    public String showLogin() {
+        return "login_signup";
+    }
+
+    @GetMapping("/verifyOTP")
+    public String verifyOtp(String text) {
+        return "otp_page";
+//        if (text.equals("1234")) {
+//            return "login_signup";
+//        } else {
+//            return "Invalid OTP";
+//        }
+    }
```

Comment:
add @RequestParam annotation for the parameter ,to get the access of the otp entered .

Reply:
Done.